README.md

```diff
@@ -270,6 +270,10 @@ Below stand further descriptions for each available (default) option :
 	// For example, "retro" would show the retro styled Apple logo on Darwin platforms.
 	// Note that the `--logo-style` argument overrides this setting.
 	"logo_style": "",
+	// Enable icons
+	// A nerd font is required to activate the icons. Otherwise, these are simply missing and a placeholder can be seen. https://github.com/ryanoasis/nerd-fonts
```

Comment:
```suggestion
	// A nerd font is required to activate the icons. Otherwise, these are simply missing and a placeholder can be seen.
	// You can also refer to : <https://github.com/ryanoasis/nerd-fonts>.
```